lib/src/test/resources/compileContracts.sh

```diff
@@ -1,47 +1,117 @@
 #!/bin/bash
 
+SRC_V0="src_v0"
+SRC_V1="src_v1"
+SRC_V2="src_v2"
+
+COMPILE_V0="compiled_v0"
+COMPILE_V1="compiled_v1"
+COMPILE_V2="compiled_v2"
+
+COMPILER_BIN_V1="cairo_compilers_v1"
+COMPILER_BIN_V2="cairo_compilers_v2"
```

Comment:
It is unnecessary in my opinion - you can put all compilers to `compilers` directory, and name subdirectories by compilers versions, so eg:

`(...)/compilers/2.0.0/(...)`
`(...) /compilers/1.1.1/(...)`

Reply:
Then would have to update path `src/test/resources/cairo_compilers_v2/cairo/bin/starknet-sierra-compile` in `devnetProcessBuilder` each time, so we could agree on `compilers/v1`, `compilers/v2`?

Reply:
yeah it's fine.